<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarkerView</class>
 <widget class="QMainWindow" name="MarkerView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0" colspan="3">
       <widget class="QTableView" name="tableView_CourseSelection">
        <property name="minimumSize">
         <size>
          <width>7</width>
          <height>462</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineEdit_searchField"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>Course Name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Course Number</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Professor ID</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="5" colspan="3">
       <widget class="QTableView" name="tableView_Activities"/>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TA View</string>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QComboBox" name="comboBox_contextMenu">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>25</height>
         </size>
        </property>
        <property name="iconSize">
         <size>
          <width>21</width>
          <height>22</height>
         </size>
        </property>
        <property name="duplicatesEnabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Modify Activity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DeleteActivity</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_searchCourses">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="5" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Logged in as:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_userAccountName">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="6">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Activities</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="pushButton_markSubmission">
        <property name="minimumSize">
         <size>
          <width>163</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Mark Submission</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLogout"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
